libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -67,4 +79,16 @@ public void setFeatureSetFromList(List<String> featureSet) {
         this.featureSet =
                 (featureSet != null && !featureSet.isEmpty()) ? String.join(",", featureSet) : null;
     }
+
+    public void pushState(IpvJourneyTypes journeyType, String state) {
```

Comment:
If we plan to keep this alongside `setState` for a while, we should probably move the `setState` call in here and make it private - so you can't accidentally take them out of sync - but if it's just temporary then it doesn't matter so much.

Reply:
I think I follow - but yeah this is temporary. I'm planning on removing `userState` from the session in a follow up PR. Just got to stage it out